include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -143,7 +143,7 @@ namespace picongpu
                 // openPMD
                 __getTransactionEvent().waitForFinished();
                 openPMDSeries = std::unique_ptr<::openPMD::Series>(
-                    new ::openPMD::Series(fullName, at, communicator, jsonMatcher->get("")));
+                    new ::openPMD::Series(fullName, at, communicator, jsonMatcher->getDefault()));
```

Comment:
Not a big deal, but perhaps `std::make_unique` would be slightly bettter than `std::unique_ptr<...>(new ...)`

Reply:
That is still a leftover from C++11 times, but I can take the chance to update that now.